src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -176,13 +188,30 @@ class PrebidAdUnit {
 		this.code = advert.id;
 		this.bids = bids(advert.id, slot.sizes, pageTargeting);
 		this.mediaTypes = { banner: { sizes: slot.sizes } };
+		if (isInGpidPrebidTest) {
```

Comment:
My understanding from the ticket was that we were releasing the gpid change and then holding back 2% as a test to not have it to measure the revenue uplift?

Reply:
Yep! So the intention is: 98% rolled out and 2% reserved for the A/B test control and variant